tests/components/automation/test_mqtt.py

```diff
@@ -79,7 +79,7 @@ async def test_if_fires_on_topic_and_payload_match(hass, calls):
 
     async_fire_mqtt_message(hass, "test-topic", "hello")
     await hass.async_block_till_done()
-    assert 1 == len(calls)
+    assert len(calls) == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5456697)